scripts/queued-builds-check.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-console */
+const moment = require('moment');
+const { Op } = require('sequelize');
+const { Build } = require('../api/models');
+const Mailer = require('../api/workers/Mailer');
+
+const [,, EMAIL_TO, MINUTES] = process.argv;
+const BUILD_QUEUED_MINUTES = MINUTES || 3;
+const EMAIL_SUBJECT = `ALERT ${process.env.CF_SPACE}: Site Builds Still Queued`;
+
+async function checkQueuedBuilds() {
+  const date = new Date();
+  const now = moment(date);
+  const buildQueueTime = now.clone().subtract(BUILD_QUEUED_MINUTES, 'minutes');
+
+  const options = {
+    where: {
```

Comment:
If you just want the ids you can specify `attributes: ['id'],` right above `where`.